gravitee-apim-console-webui/src/entities/application-general/application-general.fixture.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ApplicationDetails } from '../../management/application/details/general/general-ng/applicationDetails';
+
+export function fakeGeneralList(attributes?: Partial<ApplicationDetails>): ApplicationDetails {
```

Comment:
```suggestion
export function fakeApplicationDetails(attributes?: Partial<ApplicationDetails>): ApplicationDetails {
```
This is not a list it's juste One ApplicationDetails

Reply:
Done